signer/apk/jar.go

```diff
@@ -19,7 +19,8 @@ import (
 // consts and vars for formatFilename
 const (
 	maxLineByteLen      = 72
-	maxContinuedByteLen = 70 // -1 for leading space and -1 for trailing \n newline
+	maxContinuedByteLen = 70    // -1 for leading space and -1 for trailing \n newline
+	maxHeaderBytes      = 65535 // max length for wrapped / multiline headers is 2 << 15 - 1
```

Comment:
wut? 😱

Reply:
:woman_shrugging: yeah hopefully we never hit that #147 https://searchfox.org/mozilla-central/source/security/apps/AppSignatureVerification.cpp#304